samples/batch/transcription-enabled-storage/CreateByTimerFunction/CreateTranscriptionByTimer.cs

```diff
@@ -0,0 +1,348 @@
+﻿// <copyright file="CreateTranscriptionByTimer.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace CreateByTimerFunction
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Connector;
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Core;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    public static class CreateTranscriptionByTimer
+    {
+        private const int MaxMessagesPerRequest = 500;
+
+        private const int MaxFilesPerTranscriptionJob = 500;
+
+        private static DateTime StartDateTime;
+
+        private static ILogger Logger;
+
+        [FunctionName("CreateTranscriptionByTimer")]
+        public static async Task Run([TimerTrigger("0 */2 * * * *")] TimerInfo myTimer, ILogger log)
+        {
+            if (log == null)
+            {
+                throw new ArgumentNullException(nameof(log));
+            }
+
+            if (myTimer == null)
+            {
+                throw new ArgumentNullException(nameof(myTimer));
+            }
+
+            Logger = log;
+            StartDateTime = DateTime.UtcNow;
+            log.LogInformation($"C# Timer trigger function v3 executed at: {StartDateTime}. Next occurrence on {myTimer.Schedule.GetNextOccurrence(StartDateTime)}.");
+
+            if (!string.IsNullOrEmpty(CreateTranscriptionEnvironmentVariables.SecondaryLocale) && !CreateTranscriptionEnvironmentVariables.SecondaryLocale.Equals("None", StringComparison.OrdinalIgnoreCase))
```

Comment:
I do not really understand why LID is needed here.

Reply:
I just added it based on a customer demand - I'm not a big fan of how it is in the project atm either. But since there is no other way since there is no other service for LID. If you want, I can remove it before merging - otherwise I'll just keep it as an optional feature.